tools/cldr-code/src/main/resources/org/unicode/cldr/json/JSON_config_subdivisions.txt

```diff
@@ -4,7 +4,7 @@
 package=subdivisions ; packageDesc=Translated versions of locale display name elements: subdivisions.
 
 # mapping paths to sections
-section=subdivisions ; path=//cldr/subdivisions/[^/]++/localeDisplayNames/subdivisions/.* ; package=subdivisions
+section=subdivisions ; path=//cldr/subdivisions/localeDisplayNames/subdivisions/.* ; package=subdivisions
```

Comment:
What was the reason for `[^/]++/`?

Reply:
The original path was one more level deep.